packages/app/providers/auth-provider.tsx

```diff
@@ -217,6 +218,13 @@ export function AuthProvider({
     };
   }, [doRefreshToken]);
 
+  useEffect(() => {
+    if (authenticationStatus === "AUTHENTICATED") {
```

Comment:
we do it here instead of login handler. Not sure if it's a good idea. Looks safe for now.